pkgs/development/ocaml-modules/capnp/default.nix

```diff
@@ -0,0 +1,50 @@
+{ pkgs
+, lib
+, buildDunePackage
+, fetchFromGitHub
+, capnproto
+, ocplib-endian
+, res
+, result
+, stdint
+, stdio
+}:
+
+buildDunePackage rec {
+  pname = "capnp";
+  version = "3.6.0";
+
+  minimalOCamlVersion = "4.08";
+
+  src = fetchFromGitHub {
+    owner = "capnproto";
+    repo = "capnp-ocaml";
+    rev = "v${version}";
+    hash = "sha256-G4B1llsHnGcuGIarDB248QMaRBvS47IEQB5B93wY7nA=";
+  };
+
+  nativeBuildInputs = [
+    capnproto
+  ];
+
+  buildInputs = [
+    result
+    stdio
+  ];
+
+  propagatedBuildInputs = [
+    ocplib-endian
+    res
+    stdint
+  ];
+
+  doCheck = false;
```

Comment:
This line without a comment is surprising. Are tests costly, broken, irrelevant?

Reply:
Hmm I'm not sure why. I think these tests seemed a little difficult to get working.

But they weren't -- looks like enabling them wasn't so bad!
They seem like they're pretty fast, too.